app/events/alerts/limit_reached_creatable.rb

```diff
@@ -3,9 +3,11 @@ def create(alert)
     provider = alert.cinstance.provider_account
 
     new(
-      alert:    alert,
       provider: provider,
-      service:  alert.cinstance.service,
+      application_id: alert.cinstance.application_id,
```

Comment:
Why was it a problem passing an `alert` instance, while other objects are still passed as instances (e.g. `provider`)?

Reply:
Following up my previous answer. `alert` here was referencing the model in the DB, but that only existed for web alerts, not for mail alerts. As you pointed above, I removed the condition that prevented this event to be created for email alerts, so after the change we could end up creating an event which `alert` is in memory but not persisted. If you pass the unpersisted alert to the event, when the worker reads this event it will try to load the alert and fail. That's why I'm just passing the relevant info directly.